api/specs/webserver/openapi.yaml

```diff
@@ -218,6 +219,9 @@ paths:
   /projects/{study_uuid}/tags/{tag_id}:
     $ref: "./openapi-projects.yaml#/paths/~1projects~1{study_uuid}~1tags~1{tag_id}"
 
+  /projects/{study_uuid}/folder/{folder_id}:
```

Comment:
This entry is IMO unnecessary (see explanation below)

NOTE: ``study_uuid`` should not be used but instead try using ``project_uuid`` (we want to unify all these identifiers)

Reply:
```study_uuid```s replaced by ```project_uuid```s